packages/app/components/onboarding/select-social.tsx

```diff
@@ -64,35 +50,60 @@ export const SelectSocial = () => {
           </Text>
         </View>
         <View tw="mt-16 flex flex-grow-0">
-          <Button size="regular" onPress={handleConnectTwitter}>
-            <View tw="mr-1">
-              <Twitter color="#4A99E9" width={20} height={20} />
-            </View>
-            Connect Twitter{" "}
-            {loading === "twitter" ? (
-              <View tw="absolute right-4 scale-75 justify-center">
-                <Spinner size="small" color="darkgrey" />
-              </View>
-            ) : (
-              <></>
-            )}
-          </Button>
-          <Button size="regular" tw="mt-2" onPress={handleConnectInstagram}>
-            <View tw="mr-1.5">
-              <InstagramColorful width={20} height={20} />
-            </View>
-            Connect Instagram{" "}
-            {loading === "instagram" ? (
-              <View tw="absolute right-4 scale-75 justify-center">
-                <Spinner size="small" color="darkgrey" />
-              </View>
-            ) : (
-              <></>
-            )}
-          </Button>
+          <ConnectButton
+            title="Connect Twitter"
+            icon={<Twitter color="#4A99E9" width={20} height={20} />}
+            type="twitter"
+          />
+          <ConnectButton
+            title="Connect Instagram"
+            icon={<Instagram color="#4A99E9" width={20} height={20} />}
+            type="instagram"
+          />
```

Comment:
The color here is wrong. We have a new `InstagramColorful` Icon which does not need any color.

Reply:
Done, thanks!